### DISCUSSION TITLE
Discussion teste 1

### DISCUSSION THREAD

**Discussion by danilaafonso** [danilaafonso/learn-github#2] [Announcements]
### Programa de Mentoria Github 4 Women

Em parceria com a Microsoft, a WoMakersCode desenvolveu um programa de mentoria para aprendizado de Github. As aulas possuem transmissão ao vivo pelo Youtube às terça e quinta-feira. Os links são enviados por e-mail para os alunos. O programa possue uma trilha de aprendizado completa com atividades, avaliações práticas e vídeos interativos. As aulas serão ministradas durante 5 semanas e os melhores alunos serão premiados com um voucher para realizar a prova de certificação. 

![Captura de tela 2024-06-03 234002](https://github.com/danilaafonso/learn-github/assets/169950639/c0e24e10-fd47-4f46-b619-034ad45ebe46)

****Pessoal, poderiam inserir um comentário para eu finalizar a atividade?? Obrigada :)
@danilaafonso @helenformighieri @daninere @thatimoreira @leticosta4 @tthayza @anamariasilva @joselainejrs @pilardib

**Comment by tthayza** [comment]
Está sendo incrível aprender mais sobre GitHub com vocês! 🦋

**Reply by danilaafonso** [reply]
### Obrigada pela ajuda Thayza ♥

**Comment by helenformighieri** [comment]
Esta sendo uma jornada incrível!

**Reply by danilaafonso** [reply]
Obrigada Helen ♥

**Comment by pilardib** [comment]
tem sido uma experiência incrível! estou aprendendo muito

**Reply by danilaafonso** [reply]
Obrigada Pilar ♥

**Comment by danilaafonso** [comment]
### Obrigada meninas pela ajuda♥♥
